stdlib/contrib/jsternberg/aggregate/aggregate.flux

```diff
@@ -39,6 +39,32 @@ import "contrib/jsternberg/math"
 //     ])
 builtin table : (<-tables: [A], columns: C) => [B] where A: Record, B: Record, C: Record
 
+// window will aggregate columns and create tables by
+// organizing incoming points into windows.
+//
+// Each table will have two additional columns: start and stop.
+// These are the start and stop times for each interval.
+// It is not possible to use start or stop as destination column
+// names with this function. The start and stop columns are not
+// added to the group key.
+//
+// The same options as for table apply to window.
+// In addition to those options, window requires one
+// additional parameter.
+//     every = duration
+//         The duration between the start of each interval.
+//
+// Along with the above required option, there are a few additional
+// optional parameters.
+//     time = string
+//         The column name for the time input.
+//         This defaults to _time or time, whichever is earlier in
+//         the list of columns.
+//     period = duration
+//         The length of the interval. This defaults to the
+//         every duration.
+builtin window : (<-tables: [A], ?time: string, every: duration, ?period: duration, columns: C) => [B] where A: Record, B: Record, C: Record
```

Comment:
I guess if the time column must be in order, then a call to aggregate.window should be preceded by a call to sort.

Reply:
Yea but it's left up to the user to deal with this. Since the planner doesn't have a way of removing a sort from already sorted data, I didn't want to add it.

I've been thinking about this though and I think I might try to change this to be more relaxed about the time ordering since it's really inconvenient to use this with `group()`.

Reply:
Yeah, I think as long as it is documented clearly, then users can be expected to add their own sort. If a lot of folks run into issues around this, then maybe there can be a future update or something.